packages/content/src/content.ts

```diff
@@ -86,21 +105,31 @@ export function getContent(
       return Promise.reject(e);
     });
   }
-  return getContentPromise.then(content => {
-    // see if there are additional properties to fetch based on content type
-    const propertiesToFetch: IContentPropertyRequests = {};
-    if (shouldFetchData(content.hubType)) {
-      propertiesToFetch.data = getContentData;
-    }
-    if (shouldFetchOrgId(content)) {
-      propertiesToFetch.orgId = getOwnerOrgId;
-    }
-    if (Object.keys(propertiesToFetch).length === 0) {
-      // nothing more to fetch
-      return content;
-    }
-    return _fetchContentProperties(propertiesToFetch, content, options);
-  });
+  return getContentPromise.then(content => enrichContent(content, options));
+}
+
+/**
+ * Get content either from an IModel or an ID.
+ * @param idOrModel - An IModel (with our without data), or Hub API slug ({orgKey}::{title-as-slug} or {title-as-slug})
+ * or record id ((itemId}_{layerId} or {itemId})
+ * @param options - request options that may include authentication
+ */
+export function getContent(
+  idOrModel: string | IModel,
```

Comment:
Could scope this down to just an `IModel` for next major version bump if we go further down the 2-path route @mikeringrose was talking about in the issue.